bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/pp_H_TTG_012j_5f_NLO_MidDM/pp_H_TTG_012j_5f_NLO_MidDM_madspin_decay.dat

```diff
@@ -0,0 +1,28 @@
+#************************************************************
+#*                        MadSpin                           *
+#*                                                          *
+#*    P. Artoisenet, R. Frederix, R. Rietkerk, O. Mattelaer * 
+#*                                                          *
+#*    Part of the MadGraph5_aMC@NLO Framework:              *
+#*    The MadGraph5_aMC@NLO Development Team - Find us at   *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#*    Manual:                                               *
+#*    cp3.irmp.ucl.ac.be/projects/madgraph/wiki/MadSpin     *
+#*                                                          *
+#************************************************************
+#Some options (uncomment to apply)
+#
+# set seed 1
+# set Nevents_for_max_weight 75 # number of events for the estimate of the max. weight
+# set BW_cut 15                 # cut on how far the particle can be off-shell
+set spinmode none          # Use one of the madspin special mode
+set max_weight_ps_point 400  # number of PS to estimate the maximum for each event
+
+#import model DipoleMomentMid_UFO models/DipoleMomentMid_UFO/param_test.dat --bypass_check
+#import model DipoleMomentMid_UFO models/DipoleMomentMid_UFO/param_signal.dat --bypass_check
+import model DipoleMomentMid_UFO ./param_signal.dat --bypass_check #unsure what to specify param_signal.dat relative to
+
+decay h > ta+ ta- a
+
+launch
```

Comment:
please change the card to following (one question, i never use the syntax like you, where did you find the reference):

set ms_dir ./madspingrid

#initialization parameters
set Nevents_for_max_weight 250 # number of events for the estimate of the max. weight
set max_weight_ps_point 400  # number of PS to estimate the maximum for each event

set max_running_process 1

decay h > ta+ ta- a

launch

Reply:
Between the card at this link and the other comment saying to import the model before I define protons and jets, I'm confused where I should be importing my model. Per this post on launchpad, I'm currently importing the model in the madspin card and not in the proc card: https://answers.launchpad.net/mg5amcnlo/+question/703508